share/spice/libraries/footer.handlebars

```diff
@@ -0,0 +1,6 @@
+{{#if latest_release}}
+<div>{{latest_release}} Latest Release</div>
+{{else}}
+<div>&nbsp;</div>
```

Comment:
I think you can skip the empty div? Removing it from the DOM doesn't seem to have any effect